7_react/react1/week1/todo-app/src/App.js

```diff
@@ -0,0 +1,26 @@
+
+import './App.css';
+ 
+export default function TodoListApp (){
```

Comment:
Consider to give the method the same name as the component or file. As developer I will expect `App.js` to export a function named `App()`